install_scripts/templates/common/kubernetes.sh

```diff
@@ -551,6 +551,19 @@ k8sNamespaceExists() {
     kubectl get namespaces | grep "$1" > /dev/null
 }
 
+#######################################
+# Return status code 0 if the storageclass for the provisioner is the default, else 1
+# Globals:
+#   None
+# Arguments:
+#   namespace
+# Returns:
+#   None
+#######################################
+defaultStorageClassForProvisionerExists() {
+    kubectl get storageclass -o=jsonpath='{.items[?(@.provisioner=="$1")].metadata.annotations}' | grep "storageclass.kubernetes.io/is-default-class":"true" > /dev/null
```

Comment:
`?(@.provisioner=="$1")` what does this mean?
Also, grep -q is a bit cleaner than grep > /dev/null

Reply:
1. filters for the provisioner, e.g. `ceph.rook.io/block`
2. 👍 , thanks for the tip